src/branch_view.js

```diff
@@ -4,8 +4,8 @@ goog.require('utils');
 goog.require('banner_css');
 goog.require('safejson');
 
-function renderHtmlBlob(parent, html, bannerElement) {
-	var re, match;
+function renderHtmlBlob(parent, html, hasApp) {
+	var re, match, ctaText;
```

Comment:
The defaults are currently set on lines 19 and 25. Do you think it makes more sense to set them at the top though?